lighthouse-core/closure/closure-type-checking.js

```diff
@@ -48,25 +48,24 @@ gulp.task('compile-report', () => {
     warning_level: process.env.CI ? 'QUIET' : 'VERBOSE',
     jscomp_error: [
       'checkTypes',
+      'missingProperties',
+      'accessControls',
+      'const',
+      'visibility',
+      'reportUnknownTypes',
```

Comment:
add a comment here saying we can move reportUnknownTypes back to warning if necessary?